src/containerapp/azext_containerapp/containerapp_job_decorator.py

```diff
@@ -278,7 +276,7 @@ def construct_payload(self):
         eventTriggerConfig_def = None
         if self.get_argument_trigger_type is not None and self.get_argument_trigger_type().lower() == "event":
```

Comment:
`self.get_argument_trigger_type` should be `self.get_argument_trigger_type()`

Reply:
Hi @p-bouchon 
Can you help to change `self.get_argument_trigger_type` to `self.get_argument_trigger_type()` in line 279 and line 271?
And rerun all tests for job.